frontend/src/components/common/cards/SummaryCard.vue

```diff
@@ -0,0 +1,81 @@
+<script setup lang="ts">
+import { ref, type PropType, onMounted } from 'vue';
+import Card from 'primevue/card';
+import Button from 'primevue/button';
+import type { FamUserRoleAssignmentCreate } from 'fam-api';
+import {
+    selectedApplicationDisplayText,
+    useNotificationMessage,
+} from '@/store/ApplicationState';
+
+const selected = ref('');
+const props = defineProps({
+    data: {
+        type: Object as PropType<FamUserRoleAssignmentCreate>,
+        required: true,
+    },
+});
+
+onMounted(() => {
+    console.log('I am here', props.data);
+    useNotificationMessage.notificationMsg = `New access granted to ${props.data.user_name}`;
+});
+</script>
+<template>
+    <div class="row">
+        <div class="col-6">
+            <Card class="mb-2 p-0 p-card">
+                <template #title>Summary</template>
+                <template #subtitle>Review your access request</template>
+                <template #content>
+                    <div class="card-content">
+                        <p>
+                            <label>User name:&nbsp;</label
+                            ><span>{{ props.data.user_name }}</span>
+                        </p>
+                        <p v-if="props.data.forest_client_number">
+                            <label>Forest Client ID:&nbsp;</label
+                            ><span>
+                                {{ props.data.forest_client_number }}
+                            </span>
+                        </p>
+                        <p>
+                            <label>Application:&nbsp;</label
+                            ><span>{{ selectedApplicationDisplayText }} </span>
+                        </p>
+                    </div>
+                </template>
+                <template #footer>
+                    <div class="card-footer">
+                        <Button
+                            type="submit"
+                            id="grantAccessSubmit"
+                            class="mb-3"
+                            label="Submit"
+                            v-on:click="$emit('submit')"
+                        ></Button>
```

Comment:
Does our design of SummaryCard needs to include "submit" button?
Also, this SummaryCard looks to me is quite specific for displaying ONLY for grantAccess data and the button id even says "grantAccessSubmit". Initially I thought "Summary"Card, is a "common" component that it may be used in different "summary"....